php80/fpm/conf/php.ini

```diff
@@ -5,7 +5,7 @@ memory_limit = 512M
 log_errors = 1
 error_reporting = E_ALL
 
-xdebug.remote_enable = 1
```

Comment:
This setting has been replaced by `xdebug.mode` with value `debug`.
- https://xdebug.org/docs/upgrade_guide